lib/LoginStrategy.ts

```diff
@@ -334,4 +340,39 @@ export class LoginStrategy extends BaseStrategy {
     }
     return `${didParts[0]}:${didParts[1]}:${chainName}:${didParts[2]}`;
   }
+
+  validateTokenPayload(payload: unknown): payload is ITokenPayload {
```

Comment:
```suggestion
  isTokenPayload(payload: unknown): payload is ITokenPayload {
```

Reply:
Could you explain why this is a better name?

Reply:
It is unclear what validation mean